documentation/docs/verifiable_credentials/overview.md

```diff
@@ -1,46 +1,49 @@
+There are several types of actors that play different roles in a verifiable credential system. Starting with a common example of how things work today that you will find below, this article explains how physical credentials and centralized databases are used while outlining the roles that various entities play in the Verifiable Credential system.
 
-:::tip Example - Passport Issuance
+:::note Example - Passport Issuance
 
 A government (the _Issuer_) issues a passport asserting citizenship (the _Verifiable Credential_) to Alice (the _Subject_ and _Holder_), and writes the information to a database (the _Verifiable Data Registry_). When crossing the border, Alice (the _Holder_) presents her passport to a border agent (the _Verifier_) who can verify that Alice (the _Subject_) is indeed a citizen.
 
 :::
 
 **Subject:** An entity about which claims are made – Alice (the _Subject_) is a citizen of this country.
 
-**Holder:** An entity which posseses verifiable credentials – Alice (the _Holder_) posseses the passport (the _VC_).
+**Holder:** An entity which possesses verifiable credentials – Alice (the _Holder_) possesses the passport (the _VC_).
 
 **Issuer:** An entity which asserts claims about a subject – The governing body (the _Issuer_), which is trusted, issues Alice a passport.
 
-**Verifier:** An entity which check's if the VC a holder presents is legitimate – The border agent (the _Verifier_) trusts the government (the _Issuer_) which issued Alice her passport, and validates that Alice (the _Subject_) is a citizen.
+**Verifier:** An entity which checks if the VC a holder presents is legitimate – The border agent (the _Verifier_) trusts the government (the _Issuer_) which issued Alice her passport and validates that Alice (the _Subject_) is a citizen.
 
 :::note
 
-See the [Verifiable Credentials Data Model 1.0 Specification](https://w3c.github.io/vc-data-model/) for more information.
+For more information, you can check out the [Verifiable Credentials Data Model 1.0 Specification page](https://w3c.github.io/vc-data-model/).
 
 :::
 
 ### Verifiable Credentials in IOTA
 
-In the IOTA Identity framework, instead of a physical passport being given to Alice with the passport information being written into a centralized database owned by the government, Alice receives a digital verifiable credential, and the information required for verification in the future is written to the Tangle.
+In the IOTA Identity framework, instead of a physical passport being given to Alice with the passport information being written into a centralized database owned by the government (TW Note: I'm not sure if this phrase inside the commas is necessary), Alice receives a digitally verifiable credential and the information required for verification in the future is written to the Tangle.
```

Comment:
I think it adds something by telling the user how the process changes.  That being said, I think the paragraph could use some rephrasing or restructuring.